client/app/scripts/superdesk-archive/module.js

```diff
@@ -57,6 +57,13 @@ define([
             this.count = 0;
         };
 
+        /**
+         * Get list of selected items
+         */
+        this.getItems = function getItems() {
```

Comment:
Yeah, but don't stack traces already contain the name that is assigned to the method when it is bound to controller?